frontend/ucc_gui/src/containers/Account/Register/Register.css

```diff
@@ -1,193 +1,155 @@
+    padding-bottom: calc(0.375rem + 1px);
+    margin-bottom: 0;
+    font-size: inherit;
+    line-height: 1.5;
+    float: left;
+    color: #000;
     width: 100%;
-  }
+    /* font-weight: bold; */
+    /* width: 50%; */
 }
 
 .register-form input {
-  display: block;
-  width: 100%;
-  padding: 0.375rem 0.75rem;
-  font-size: 1.1rem;
-  line-height: 1.5;
-  color: #000;
-  background-color: #fff;
-  background-clip: padding-box;
-  border: 1px solid #ced4da;
-  border-radius: 0.25rem;
-  transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
-}
-::-webkit-input-placeholder {
-  font-size: 0.8rem;
-  color: #808080;
-}
-
-@media only screen and (max-width: 40em) {
-  .register-form input {
-    width: 85%;
-  }
-}
-
-.register-form TextField {
-  padding-top: calc(0.375rem + 1px);
-  padding-bottom: calc(0.375rem + 1px);
-  margin-bottom: 0;
-  font-size: inherit;
-  line-height: 1.5;
-  float: left;
-  color: #000;
-  width: 100%;
+    display: block;
+    width: 100%;
+    padding: 0.3rem 0.5rem;
+    font-size: 1.1rem;
+    line-height: 1.5;
+    color: #000;
+    background-color: #fff;
+    background-clip: padding-box;
+    border: 2px solid #8ddef4;
+    border-radius: 0.25rem;
+    transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+    -webkit-border-radius: 5px;
 }
 
-@media only screen and (max-width: 40em) {
-  .register-form TextField {
-    margin-left: 0px;
-    width: 100%;
-  }
+::-webkit-input-placeholder {
+    font-size: 0.8rem;
+    color: #808080;
 }
 
 .terms-checkbox {
-  margin-top: 10px;
-  float: left;
+    margin-top: 10px;
+    float: left;
 }
 
 .terms-checkbox p {
-  display: flex;
-  flex-flow: row;
+    display: flex;
+    flex-flow: row;
 }
 
 .terms-print {
-  float: right;
+    float: right;
 }
 
 .register-btn {
-  background: #1cbfec;
-  color: #ffffff;
-  padding: 10px 15px;
-  display: flex;
-  justify-content: center;
-  margin-left: auto;
-  margin-right: auto;
-  font-size: 1.2rem;
-  font-weight: bold;
-  cursor: pointer;
-  border-radius: 5px;
-  border: none;
-  -webkit-appearance: none;
-  -webkit-border-radius: 5px !important;
-}
-
-@media only screen and (max-width: 40em) {
-  .register-btn {
+    background: #1cbfec;
+    color: #ffffff;
+    padding: 10px 15px;
     display: flex;
     justify-content: center;
-    margin-right: 15px;
-    margin-top: auto;
-    width: 98%;
-  }
+    margin-left: auto;
+    margin-right: auto;
+    font-size: 1.2rem;
+    font-weight: bold;
+    cursor: pointer;
+    border-radius: 5px;
+    border: none;
+    -webkit-appearance: none;
+    -webkit-border-radius: 5px !important;
+}
+
+:root {
+    --color--text: 0, 0, 0;
 }
 
 .register-btn button:disabled {
-  background: rbg(0, 0, 0, 0.06);
+    background: rbga(var(--color--text), 0.06);
 }
 
 .register-btn button:enabled {
-  background: #1a73e8;
+    background: #1a73e8;
+}
+
+@media only screen and (max-device-width: 520px) and (max-width: 40em){
```

Comment:
Just check that the view will show up correctly for a window width of 320px also.
http://www.javascriptkit.com/dhtmltutors/cssmediaqueries2.shtml

Reply:
It works for the smaller screen.